App/src/main/java/vut/fit/gja2023/app/entity/UserBo.java

```diff
@@ -4,31 +4,33 @@
 import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Size;
 import lombok.Data;
-import vut.fit.gja2023.app.enums.Role;
+import vut.fit.gja2023.app.enums.UserRole;
 
 import java.util.List;
 
 @Entity
 @Data
-@Table(name = "user")
+@Table(name = "user", indexes = {
+    @Index(name = "idx_login", columnList = "login")
+})
 public class UserBo {
     public static final int NAME_MAX_LENGTH = 30;
     public static final String LOGIN_REGEX = "^x[a-z]{5}\\d{2}$";
-    
+
     @Id
     @GeneratedValue
     private Long id;
 
     @Size(max = NAME_MAX_LENGTH)
     @Column(name = "name", nullable = false, length = NAME_MAX_LENGTH)
     private String name;
-    
+
     @Pattern(regexp = LOGIN_REGEX, message = "Invalid login format")
-    @Column(name = "login", nullable = false)
+    @Column(name = "login", nullable = false, unique = false, length = 8) //unique = true throws exception
     private String login;
```

Comment:
Je tam nejaky problem s tym dialektom, co pozeram tak to generuje alter table ktory potom sa snazi nastavit unique constraint na uz vytvoreny table, co v sqlite neni mozne, takze je chyba v tom community dialekte... kazdopadne unique = false je default tak kludne zmaz